scss/_patterns_rule.scss

```diff
@@ -0,0 +1,20 @@
+@import 'settings';
+
+$small-offset: map-get($grid-margin-widths, small);
+$default-offset: map-get($grid-margin-widths, default);
```

Comment:
When defined like this it will pollute global context (variables will be available inside whole vanilla and wherever its imported) which is not expected from such internal aliases.

And also, you seem to need them in one place only in the file, so not really needed I think?
And if we do reuse `is-fixed-width` from `hr` as suggested in other comment they will likely not be needed at all I guess?

Reply:
Ah yes I didn't think of the scope issue! It's just that formulas like `max-width: calc($grid-max-width - 2 * map-get($grid-margin-widths, default));` can get unwieldy and hard to read very quickly!

Reply:
we just put `.u-no-max-width` in that case